pointerevents/extension/pointerevent_constructor.html

```diff
@@ -0,0 +1,66 @@
+<!doctype html>
+<html>
+    <head>
+        <title>PointerEvent: Constructor test</title>
+        <meta name="viewport" content="width=device-width">
+        <link rel="stylesheet" type="text/css" href="pointerevent_styles.css">
+        <script src="/resources/testharness.js"></script>
+        <script src="/resources/testharnessreport.js"></script>
+        <!-- Additional helper script for common checks across event types -->
+    </head>
+    <body>
+    <h1>PointerEvent: Dispatch custom event with coalesced events</h1>
+    <h4>Test Description: This test checks if PointerEvent constructor works properly with respect to the coalesced events.</h4>
+    <div id="target0"></div>
+    <script>
+        async_test(function() {
+            var target0 = document.getElementById("target0");
+            // set values for non-default constructor
+            on_event(target0, "pointermove", this.step_func(function(event) {
+                generate_tests(assert_equals, [
+                    ["getCoalescedEvents().length", event.getCoalescedEvents().length, 2],
```

Comment:
What about target and current target should we assert they are the same as the original event?